docs/app/pages/home/backend-bundles-section/backend-bundles-section.component.html

```diff
@@ -0,0 +1,248 @@
+    </li>
+    <li class="features__item">
+      <h3 class="features__title">Ready to use</h3>
+      <p class="features__description">We prepared this backend pack as development basement which lets your team concentrate on business logic and data models. Now .NET and .NET Core are available as frameworks of choice. More are in progress…</p>
+    </li>
+  </ul>
+
+  <picture class="bundle-scheme">
+    <source type="image/webp" srcset="/assets/img/bundle-scheme@1x.webp 1x, /assets/img/bundle-scheme@2x.webp 2x">
+    <img class="bundle-scheme__image" src="/assets/img/bundle-scheme@1x.png" srcset="/assets/img/bundle-scheme@2x.png 2x" alt="Backend bundles">
+  </picture>
+
+  <div class="package-switcher">
+    <div [(ngModel)]="licenseType" ngbRadioGroup
+         class="package-switcher__btn-group btn-group btn-group-toggle btn-outline-toggle-group btn-group-full-width btn-toggle-radio-group">
+      <label ngbButtonLabel class="package-switcher__btn btn btn-outline-secondary">
+        <input ngbButton type="radio" value="Personal"> Personal
+      </label>
+      <label ngbButtonLabel class="package-switcher__btn btn btn-outline-secondary">
+        <input ngbButton type="radio" value="Commercial"> Commercial
+      </label>
+    </div>
+  </div>
+
+  <ul class="packages">
+    <li class="packages__item">
+      <nb-flip-card [showToggleButton]="false" [flipped]="firstCardFlipped" (click)="firstCardFlipped = !firstCardFlipped">
+        <nb-card-front class="package-card package-card--front">
+          <nb-card>
+            <nb-card-body>
+              <div class="package-card__header">
+                <span class="package-card__demo-link">Coming soon</span>
+              </div>
+              <div class="package-card__image package-card__image--dot-net"></div>
+              <div class="package-card__price-wrapper">
+                <span class="package-card__price package-card__price--old">${{ licenseType === 'Commercial' ? '2000' : '200' }}</span>
+                <span class="package-card__price">${{ licenseType === 'Commercial' ? '1500' : '150' }}</span>
+              </div>
+              <h4 class="package-card__title">{{ firstBundleName }}</h4>
+              <p class="package-card__description">
+                E-commerce Dashboard integrated with REST data services based on .NET Framework, Web API and Entity Framework 6.2
+              </p>
+              <a [href]="firstBundleMail"
+                 target="_blank"
+                 class="package-card__buy-link btn btn-demo"
+                 nbButton
+                 (click)="stopPropagation($event)">
+                 Buy now
+              </a>
+            </nb-card-body>
+          </nb-card>
+        </nb-card-front>
+        <nb-card-back class="package-card package-card--back">
+          <nb-card>
+            <nb-card-body>
+              <h4 class="package-card__type">{{ licenseType === 'Commercial' ? 'Commercial' : 'Personal' }}</h4>
```

Comment:
You can just use `licenseType` component property.
```suggestion
              <h4 class="package-card__type">{{ licenseType }}</h4>
```
Then it's better to rename it to `selectedLicenseType` to be more accurate.